edb/lib/std/30-strfuncs.eql

```diff
@@ -91,3 +91,86 @@ std::`>` (l: std::str, r: std::str) -> std::bool
 CREATE INFIX OPERATOR
 std::`>=` (l: std::str, r: std::str) -> std::bool
     FROM SQL OPERATOR r'>=';
+
+
+CREATE FUNCTION
+std::ord(val: std::str) -> std::int64
+{
+    FROM SQL FUNCTION 'ascii';
+};
+
+
+CREATE FUNCTION
+std::chr(val: std::int64) -> std::str
+{
+    FROM SQL $$
+    SELECT chr(val::int4)
+    $$;
+};
+
+
+CREATE FUNCTION
+std::str_lower(s: std::str) -> std::str
+{
+    FROM SQL FUNCTION 'lower';
+};
+
+
+CREATE FUNCTION
+std::str_upper(s: std::str) -> std::str
+{
+    FROM SQL FUNCTION 'upper';
+};
+
+
+CREATE FUNCTION
+std::str_title(s: std::str) -> std::str
+{
+    FROM SQL FUNCTION 'initcap';
+};
+
+
+CREATE FUNCTION
+std::str_lpad(s: std::str, n: std::int64, fill: std::str=' ') -> std::str
+{
+    FROM SQL $$
+    SELECT lpad(s, n::int4, fill)
+    $$;
+};
+
+
+CREATE FUNCTION
+std::str_rpad(s: std::str, n: std::int64, fill: std::str=' ') -> std::str
+{
+    FROM SQL $$
+    SELECT rpad(s, n::int4, fill)
+    $$;
+};
+
+
+CREATE FUNCTION
+std::str_ltrim(s: std::str, tr: std::str=' ') -> std::str
+{
+    FROM SQL FUNCTION 'ltrim';
+};
+
+
+CREATE FUNCTION
+std::str_rtrim(s: std::str, tr: std::str=' ') -> std::str
+{
+    FROM SQL FUNCTION 'rtrim';
+};
+
+
+CREATE FUNCTION
+std::str_trim(s: std::str, tr: std::str=' ') -> std::str
+{
+    FROM SQL FUNCTION 'btrim';
+};
+
+
+CREATE FUNCTION
+std::str_reverse(s: std::str) -> std::str
```

Comment:
Let's have a polymorphic `reverse()` function that works on bytes, strings, and arrays.

Reply:
There's no function for reversing bytea or arrays in Postgres. How much sense does the function for reversing a string make? It seems to be a good generalization for languages that deal with iterables and sequences generically, but since we don't have these broad concents in EdgeQL, does it really make sense to reverse strings? There's also the trouble with reversing arbitrary unicode.

Reply:
Yeah, drop it.

Reply:
Dropped.